go.mod

```diff
@@ -273,8 +273,8 @@ require (
 	github.com/gertd/go-pluralize v0.2.1
 	github.com/google/go-cmp v0.6.0
 	github.com/hashicorp/aws-cloudformation-resource-schema-sdk-go v0.23.0
-	github.com/hashicorp/aws-sdk-go-base/v2 v2.0.0-beta.61
-	github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2 v2.0.0-beta.62
+	github.com/hashicorp/aws-sdk-go-base/v2 v2.0.0-beta.62
```

Comment:
May need a `make clean-tidy` to propagate into the tools subdirectory as well.

Reply:
Done.